README.md

```diff
@@ -1,2 +1,36 @@
 # To-do-List
 "To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## Built With
+- HTML
+- CSS
+- JavaScript
+# Get Started
+
+ -[Awesome Books](https://yayoamigo.github.io/)
+
+
+## Authors
+
+👤 **DavidZambrano**
+
+- GitHub: [@yayoamigo](https://github.com/yayoamigo)
+- Twitter: [@carpediem0011](https://twitter.com/carpediem0011)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/david-zambrano-corral-b87a4198/)
+
+
+
```

Comment:
[Required] please add the following section to your readme file, it's important that you provide information on how other developers or future collaborators can clone and run this project on their computers. 
kindly add the following sections to your readme file with instructions on cloning your repository and setting it up locally.
- Prerequisites
- Setup
- Install